exercises/queen-attack/src/example/java/BoardCoordinate.java

```diff
@@ -0,0 +1,39 @@
+public final class BoardCoordinate {
+
+    private final int rank;
+
+    private final int file;
+
+    public BoardCoordinate(final int rank, final int file) throws IllegalArgumentException {
```

Comment:
oooohhhh!  I missed the part where `BoardCoordinates` is in the `example` sourceset.  I absolutely agree.